addons/binding/org.openhab.binding.homematic/README.md

```diff
@@ -274,3 +274,22 @@ A device may return this failure while fetching the datapoint values. I've teste
 Fetching values is only done at startup or if you trigger a REFRESH. I hope this will be fixed in one of the next CCU firmwares.  
 With [Homegear](https://www.homegear.eu) everything works as expected.
 
+### Debugging and Tracing
+If you want to see what's going on in the binding, switch the loglevel to DEBUG in the Karaf console
```

Comment:
please add an empty line after the header as Jekyll otherwise breaks the formatting